certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json

```diff
@@ -2,6 +2,23 @@
   "basePath": "/",
   "consumes": [],
   "definitions": {
+    "Actiontype": {
+      "description": "The Actiontype of the synchronous action that you wish to execute",
```

Comment:
@GAC-AJO please make sure that newely added description and x-ms-summary properties are grammatically correct and are free of typographical errors. Concatenated strings (e.g. actiontype) may cause wrong translation and strings in CaMel case are considered non-translatable.
More information about English strings can be found in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
Changed Actiontype to action type.